src/main/scala/uk/gov/hmrc/SbtArtifactory.scala

```diff
@@ -38,25 +41,20 @@ object SbtArtifactory extends sbt.AutoPlugin {
   private lazy val maybeArtifactoryUsername   = sys.env.get(artifactoryUsernameEnvKey)
   private lazy val maybeArtifactoryPassword   = sys.env.get(artifactoryPasswordEnvKey)
 
-  val artifactoryLabsPattern = ".*lab([0-9]{2}).*".r
+  val artifactoryLabsPattern: Regex = ".*lab([0-9]{2}).*".r
 
   private lazy val maybeArtifactoryCredentials = for {
     uri      <- maybeArtifactoryUri
     username <- maybeArtifactoryUsername
     password <- maybeArtifactoryPassword
   } yield directCredentials(uri, username, password)
 
-  private val bintrayUsernameEnvKey = "BINTRAY_USERNAME"
-  private val bintrayPasswordEnvKey = "BINTRAY_PASSWORD"
-  private lazy val maybeBintrayUsername = sys.env.get(bintrayUsernameEnvKey)
-  private lazy val maybeBintrayPassword = sys.env.get(bintrayPasswordEnvKey)
-
   object autoImport {
     val unpublishFromArtifactory = taskKey[Unit]("Unpublish from Artifactory.")
     val unpublishFromArtifactoryBintrayDistribution = taskKey[Unit]("Unpublish from Artifactory bintray-distribution.")
     val unpublishFromBintray = taskKey[Unit]("Unpublish from Bintray.")
     val unpublish = taskKey[Unit]("Unpublish from Artifactory.")
-    val distributeToBintray =
+    val publishToBintray =
```

Comment:
Is it worth changing the task name? Logs and task description etc. still use "distribute"
Means we should also change the name of `publishAndDistribute` which does both?

Reply:
Previously we were calling a distribute endpoint in artifactory and now we are publishing directly to Bintray hence I thought it's worth updating to the task name to reflect this. I will update the description too.

I left `publishAndDistribute` as is as it would mean co-ordinating a jenkins seed with a release of a new version of sbt-artifactory so harder to roll back. We don't call `distributeToBintray ` directly so it should be a safe change to make.